pkgs/top-level/all-packages.nix

```diff
@@ -673,6 +673,11 @@ in
 
   bonfire = callPackage ../tools/misc/bonfire { };
 
+  buildbot = with python3Packages; toPythonApplication buildbot;
+  buildbot-ui = with python3Packages; toPythonApplication buildbot-ui;
+  buildbot-full = with python3Packages; toPythonApplication buildbot-full;
+  buildbot-worker = with python3Packages; toPythonApplication buildbot-worker;
```

Comment:
No `buildbot-full` and `buildbot`-pkg`? I must admit I have no idea what the difference is.

Reply:
`buildbot-pkg` is just a Python library. `buildbot-full` is there.

Reply:
No idea how I overlooked that :)